ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -138,6 +138,14 @@ - (NSDictionary *)constantsToExport
                                                         target:targetUrl] build]];
 }
 
+RCT_EXPORT_METHOD(setCustomDimensionValue:(NSString *)trackerId dimensionIndex:(NSInteger)dimensionIndex dimensionValue:(NSString *)dimensionValue)
+{
+    id<GAITracker> tracker = [[GAI sharedInstance] trackerWithTrackingId:trackerId];
+    
+    [tracker set:[GAIFields customDimensionForIndex:dimensionIndex] value:dimensionValue];
```

Comment:
How about my previous comment, having them optional parameters to trackScreenView and trackEvent (which already has optional parameters) ?